docs/core/diagnostics/logging-tracing.md

```diff
@@ -0,0 +1,94 @@
+---
+title: .NET Core Logging and Tracing
+description: An introduction to .NET Core Logging and Tracing.
+author: sdmaclea
+ms.author: stmaclea
+ms.date: 08/05/2019
+---
+# .NET Core Logging and Tracing
+
+Logging and tracing is a primitive technique used since the early days of computers. It simply involves instrumenting an application to produce output to be consumed later.
+
+Logging and tracing are similar. As used here we'll distinguish:
```

Comment:
We should be avoided unless Microsoft is recommending something
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/we

```suggestion
Logging and tracing are similar. In this article, these two terms are distinguished as follows:

```